electron/main.ts

```diff
@@ -101,22 +104,35 @@ app.on('activate', () => {
 });
 
 function openFile() {
```

Comment:
Before we all get confused we should probably rename this to `openDir` or something.

Reply:
📁